### PR TITLE
Fix for server test defect breaks

### PR DESCRIPTION
**Pull request by chitwansharmaa**
Changes corresponds to recent build breaks. These changes are viable even after build.gradle inclusion. 
InstallServerTest fails have been tagged as high_impact. These changes should fix those build breaks and after successful implementation of build.gradle these changes do not need to be restored. 
serverX.zip changes only include server.xml change to update from old features to updated features.

### CODE REVIEW

**Comment by chitwansharmaa** [comment]
#build

File: dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallFeatureTest.java [hunk 1]
```diff
@@ -344,7 +344,16 @@ public void testFeatureLocalRepoOverride() throws Exception {
...
+        replaceWlpProperties("21.0.0.4");
+        copyFileToMinifiedRoot("etc", "../../publish/propertyFiles/publishRepoOverrideProps/featureUtility.properties");
+
+        copyFileToMinifiedRoot("repo/com/ibm/websphere/appserver/features/features/21.0.0.4",
+                "../../publish/repo/com/ibm/websphere/appserver/features/features/21.0.0.4/features-21.0.0.4.json");
 
+        copyFileToMinifiedRoot("repo/io/openliberty/features/features/21.0.0.4",
+                "../../publish/repo/io/openliberty/features/features/21.0.0.4/features-21.0.0.4.json");
+                
+        writeToProps(minifiedRoot+ "/etc/featureUtility.properties", "featureLocalRepo", minifiedRoot + "/repo/");
```
Comment (chitwansharmaa) [inline dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallFeatureTest.java:356]: This test was failing due to connection issue with maven central and inability to fetch features.json for 21005. These changes will now fetch features.json 21.0.0.4 which already exists in the FAT.

File: dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallServerTest.java [hunk 2]
```diff
@@ -137,20 +137,51 @@ public void testInstallBlankFeatures() throws Exception {
 
 
     /**
-     * Install jsf-2.2 on its own first, then install a server.xml with jsf-2.2 and cdi-1.2. The autofeature cdi1.2-jsf2.2 should be installed along with the other features.
+     * Install osgiConsole-1.0 on its own first, then install a server.xml with osgiConsole-1.0 and eventLogging-1.0. The autofeature osgiConsole-1.0-eventLogging-1.0 should be installed along with the other features.
      * @throws Exception
      */
     @Test
     public void testInstallAutoFeatureServerXml() throws Exception {
```
Comment (chitwansharmaa) [inline dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallServerTest.java:144]: This test was consistently failing due to connection problem. In InstallFeatureTest, made changes to instead test for osgiConsole and eventLogging. For server test class, made similar changes as dependency is low for these features and to remain consistent throughout the class.

File: dev/com.ibm.ws.install.featureUtility_fat/publish/tmp/autoFeatureServerXml/server.xml [hunk 3]
```diff
@@ -3,8 +3,8 @@
 
     <!-- Enable features -->
     <featureManager>
-        <feature>cdi-1.2</feature>
-        <feature>jsf-2.2</feature>
+        <feature>osgiConsole-1.0</feature>
+        <feature>eventLogging-1.0</feature>
```
Comment (chitwansharmaa) [inline dev/com.ibm.ws.install.featureUtility_fat/publish/tmp/autoFeatureServerXml/server.xml:7]: server.xml changes to instead install updated features

File: dev/com.ibm.ws.install.featureUtility_fat/publish/tmp/usrFeaturesServerXml/server.xml [hunk 4]
```diff
@@ -4,7 +4,7 @@
     <!-- Enable features -->
     <featureManager>
 	<feature>jsp-2.3</feature>
-	<feature>usr:mpHealth-2.0</feature>
+	<feature>ssl-1.0</feature>
```
Comment (chitwansharmaa) [inline dev/com.ibm.ws.install.featureUtility_fat/publish/tmp/usrFeaturesServerXml/server.xml:7]: We no longer install mpHealth-2.0, instead switched to ssl-1.0.

**Review by bensonlatibm — APPROVED** [review]

**Pull request merged**